resources/fetchers/logging_fetcher.go

```diff
@@ -72,3 +89,18 @@ func (r LoggingResource) GetMetadata() (fetching.ResourceMetadata, error) {
 	}, nil
 }
 func (r LoggingResource) GetElasticCommonData() any { return nil }
+
+func (c ConfigResource) GetMetadata() (fetching.ResourceMetadata, error) {
+	return fetching.ResourceMetadata{
+		ID:      c.GetResourceArn(),
+		Type:    fetching.CloudConfig,
```

Comment:
Why not add it to the logging type? it is in the same cis section so it make sense some how

Reply:
IMO, The type field should represent a group of resources and be cloud/benchmark agnostic as much as possible.
`Logging` refers to a benchmark section and should be used only if we had to create a hybrid type such as the `Monitoring` type.